docs/manual/workflow-steps/pagerduty.md

```diff
@@ -0,0 +1,37 @@
+# Pager Duty Plugins (Enterprise)
```

Comment:
This looks like it only covers the Job Steps.  Let's also include the following on this page:
- Overview what PagerDuty is
- Distinguish between Job Steps and Notification Plugin options.
- Where do they get the API key?  (Note, they are different between the ones used for Job steps and ones usd for Notification steps.)